test/e2e_node/e2e_node_suite_test.go

```diff
@@ -0,0 +1,59 @@
+package e2enode
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/log"
+	"github.com/loft-sh/vcluster/test/framework"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+
+	// Enable cloud provider auth
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	// Register tests
+)
+
+var (
+	scheme = runtime.NewScheme()
+)
+
+func init() {
+	_ = clientgoscheme.AddToScheme(scheme)
+	// API extensions are not in the above scheme set,
+	// and must thus be added separately.
+	_ = apiextensionsv1beta1.AddToScheme(scheme)
+	_ = apiextensionsv1.AddToScheme(scheme)
+	_ = apiregistrationv1.AddToScheme(scheme)
+}
+
+// TestRunE2ETests checks configuration parameters (specified through flags) and then runs
```

Comment:
Super minor mistake :) 
```suggestion
// TestRunE2ENodeTests checks configuration parameters (specified through flags) and then runs
```